appinventor/blocklyeditor/src/exportBlocksImage.js

```diff
@@ -304,6 +304,7 @@ function PNG() {
  * @const
  */
 PNG.HEADER = [0x89, 0x50, 0x4E, 0x47, 0x0D, 0x0A, 0x1A, 0x0A];
+var pHY_data =  [0, 0, 16, 25, 0, 0, 16, 25, 1];
```

Comment:
These values should be prefixed with `0x` as they are intended to be hex, not decimal.

Reply:
Fixed